deploy/clowdapp.yaml

```diff
@@ -149,6 +163,10 @@ objects:
             value: http://ccx-insights-content-service:${CONTENT_SERVICE_PORT}/api/v1/
           - name: CCX_NOTIFICATION_SERVICE__DEPENDENCIES__CONTENT_ENDPOINT
             value: content
+          - name: CCX_NOTIFICATION_SERVICE__DEPENDENCIES__TEMPLATE_RENDERER_SERVER
+            value: http://localhost:8083/
```

Comment:
use a parameter here please, otherwise it will be very difficult to distinguish between stage, prod and ephemeral envs. Use `{TEMPLATE_RENDERER_SERVER}` instead

Reply:
Replaced

Reply:
Replaced